code/Controllers/SilverStripeNavigatorItem_Unversioned.php

```diff
@@ -5,11 +5,30 @@
 use SilverStripe\CMS\Controllers\SilverStripeNavigatorItem;
 use SilverStripe\Core\Config\Config;
 use SilverStripe\Core\Convert;
+use SilverStripe\Dev\Deprecation;
+use SilverStripe\ORM\CMSPreviewable;
```

Comment:
Why has `CMSPreviewable` been added?

Reply:
For strong typing ` __constructor(CMSPreviewable $record)` argument